spec/integration/task_cache_spec.rb

```diff
@@ -54,4 +54,14 @@
     run_cli(%w[task show] + flags)
     expect(original_mtime).not_to eq(File.mtime(project.task_cache_file))
   end
+
+  it 'clears existing cache with --clear-cache' do
+    run_cli(%w[module generate-types] + flags)
+    original_mtime = File.mtime(project.task_cache_file)
+    run_cli(%w[task show] + flags) # copied from above test to ensure
```

Comment:
Same here, can remove this `run_cli` and the `expects` and all of the `mtime` checks.